docs/syntax.md

```diff
@@ -359,6 +359,7 @@ Makes specifications conditional. This means that they are only verified if anot
 
 ```yaml
 lifestage:
+  empty: True
```

Comment:
Remark that we need this `empty: True`, as there is an implicit `empty: False` for all terms and within the `if` condition, empty values are actually possible.